test/read_only_api_tests.jl

```diff
@@ -116,6 +116,20 @@ state_param = Dict("state" => "all")
 @test get(issue(ghjl, 40; auth = auth).title) == "Needs test"
 @test hasghobj(40, first(issues(ghjl; auth = auth, params = state_param)))
 
+#########
+# Gists #
+#########
+
+kc_gists, page_data = gists("KristofferC", page_limit=1, params=Dict("per_page" => 5))
+@test typeof(kc_gists) == Vector{Gist}
+@test length(kc_gists) != 0
+@test get(get(kc_gists[1].owner).login) == "KristofferC"
+
+gist = gist("0bace7cc774df4b3a4b0ee9aaa271ef6")
+@test length(get(gist.files)) == 2
```

Comment:
Hm, as it is right now it requires me to create a new file for this to change. If I make the revision, I can make this to change by just deleting the gist. So there is no extra "security" by specifying a revision?